fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -811,6 +841,11 @@ void MultiVersionDatabase::DatabaseState::cancelConnections() {
 
 // MultiVersionApi
 
+bool MultiVersionApi::apiVersionAtLeast(int minVersion) {
+	ASSERT(MultiVersionApi::api->apiVersion != 0);
+	return MultiVersionApi::api->apiVersion >= minVersion;
```

Comment:
I believe the implementation in `DatabaseContext` treats -1 as the maximum version, so `apiVersionAtLeast(anything)` will always work if the API version is negative. Does this need to also replicate that behavior?

Reply:
I think the reason we do that is for things like simulation. We do use the multi-version client in simulation, but we also require that the API version be set when using it. I'm not certain what version actually gets specified though, so maybe it's safest to add the same logic?

Reply:
Yeah, I think it might make sense, just to be consistent. Hm.